scripts/build_docs-api.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
I'm confused by this file. It seems identical to [build_doc.sh](https://github.com/apple/coremltools/blob/master/scripts/build_docs.sh) which we already have. Did you just want to rename that file?

Reply:
The `build_doc.sh` file uses the hard-coded `cd docs` command before doing `make docs`. Since I am using the `docs-api` directory (for several reasons, including it's a better directory name for the API docs, which are separate from other docs), I had to change just that one line of code to `cd api-docs`. This is not elegant; perhaps we should edit the original version to include an option for specifying the "docs" directory.

Reply:
After discussing with @aseemw I think the best thing to do is not include the `build-docs` script in this PR, so I am removing this. If I use my version locally, I can generate the HTML. When we replace the original `docs` folder with this new `docs-api `folder in another PR after this one merges, we don't have to change anything.